### PR TITLE
feat(engine): multiple paths on `--path` flag

### LINKED ISSUE #3001: Change flag `--path` to accept multiple paths
**Is your feature request related to a problem? Please describe.**
KICS does not allow to define multiple paths to be scanned at once, making necessary to run KICS multiple times, which is frustrating.

**Describe the solution you'd like**
Change flag `--path` to allow multiple paths, like `--exclude-path`.

**Describe alternatives you've considered**
As described, running kics multiple times on required paths.

### PR DESCRIPTION
**Pull request by felipe-avelar**
Closes #3001

**Proposed Changes**
- KICS now accepts multiple paths on -p flag
- Some docs update

**Considerations**
- If -p has more than one value, `kics.config` will not be loaded automatically, configuration flag must be explicit used to load any configuration file.

I submit this contribution under Apache-2.0 license.

### CODE REVIEW

File: docs/configuration-file.md [hunk 1]
```diff
@@ -174,4 +174,18 @@ kics scan --config kics-config.json
 kics scan
 ```
 
-**Note**: CLI flags will have priority over the configuration file properties!
+**Note**: If more than one path is given, KICS will warn that `--config` must be used to explicit decide
```
Comment (rogeriopeixotocx) [inline docs/configuration-file.md:177]: great work

**Review by rogeriopeixotocx — CHANGES_REQUESTED** [review]
this looks great, can we have just 1 E2E test before merging?

**Review by joaoReigota1 — APPROVED** [review]
LGTM

**Review by rogeriopeixotocx — APPROVED** [review]
LGTM

**Pull request merged**